dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -16,8 +16,50 @@
 
 # ACME configuration
 
-#configParm1=Configuration parameter 1 for ACME
-#configParm1.desc=Description for parameter 1
+#directoryURI=ACME server directory URI
+#directoryURI.desc=The URI to the ACME server's directory object.
 
-#configParm2=Configuration parameter 2 for ACME
-#configParm2.desc=Description for parameter 2
+#domain=Domain name
+#domain.desc=A domain name to request a certificate for.
+
+#validFor=Valid for
+#validFor.desc=The duration of time the certificate signing request will request the certificate to be valid for. The default is defined by the CA server.
+
+#country=Country
+#country.desc=Country name to use for the certificate.
+
+#state=State or province
+#state.desc=State or province name to use for the certificate.
+
+#locality=Locality
+#locality.desc=Locality name to use for the certificate.
+
+#organization=Organization
+#organization.desc=Organization name to use for the certificate.
+
+#organizationalUnit=Organizational unit
+#organizationalUnit.desc=Organizational unit name to use for the certificate.
+
+#acceptTermsOfService=Accept Terms of Service
+#acceptTermsOfService.desc=Indicates whether the user accepts the terms of service for the ACME CA. If true, the terms of service will be automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate will fail.  
+
+#challengeRetries=Challenge retries
+#challengeRetries.desc=The number of times to retry updating the challenge status before aborting the challenge.
+
+#challengeRetryWait=Challenge retry wait
+#challengeRetryWait.desc=The amount of time to wait before retrying to refresh the status of a challenge.
+
+#orderRetries=Order retries
+#orderRetries.desc=The number of times to retry updating the order status before aborting the order.
+
+#orderRetryWait=Order retry wait
+#orderRetryWait.desc=The amount of time to wait before retrying to refresh the status of an order.
+
+#accountContact=Account contact
+#accountContact.desc=A contact URL the ACME server can use to contact the client for issues related this the ACME account.
+
+#accountKeyFile=Account key file
+#accountKeyFile.desc=A path to the file containing a key identifier for a registered account on the ACME CA server. If the file does not exist a new account will be registered with the ACME CA server and the associated key will be written to this file. This file should be backed up to maintain control of the account on the ACME CA server. 
+
+#domainKeyFile=Domain key file
+#domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist a new key will be generated and written to this file. This file should be backed up to maintain control of the domain. 
```

Comment:
**change**
If the file does not exist a new key will be generated and written to this file. This file should be backed up to maintain control of the domain. 
**to**
If the file does not exist, a new key is generated and written to this file. Back this file up to maintain control of the domain.

Reply:
Made the requested changes.